app/src/main/java/com/example/eric/pokedex/FilterActivity.java

```diff
@@ -0,0 +1,78 @@
+package com.example.eric.pokedex;
+
+import android.content.Intent;
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+import android.view.View;
+import android.widget.Button;
+import android.widget.EditText;
+import android.widget.ImageView;
+
+import com.bumptech.glide.Glide;
+
+public class FilterActivity extends AppCompatActivity {
```

Comment:
would've been nicer had this been implemented as part of a menu instead of a separate activity